notebooks/advanced_algorithms/Quantum_Principal_Component_Analysis.ipynb

```diff
@@ -0,0 +1,1514 @@
+{
```

Comment:
<ul><li>I think the parenthetical in the first sentence should be its own sentence.</li><li>"Starting to be think"? Needs a rephrase.</li><li>Might be good to include a bit more explanation about why people think quantum computing is an appropriate tool for these problems.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws-samples/amazon-braket-algorithm-library/pull/118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>